tests/generator/generator.cpp

```diff
@@ -936,6 +936,41 @@ TEST(Generator, cellmlMappingsAndEncapsulations)
     EXPECT_EQ(fileContents("generator/cellml_mappings_and_encapsulations/model.py"), generator->implementationCode());
 }
 
+TEST(Generator, cellmlUnitScaling)
```

Comment:
I think we could start with a simpler test to show working units scaling.

Reply:
Fair point, not least since this is how I actually came up with this test in the end, i.e. step by step. Ok, I will go back to my commit log and split this test in several sub ones, following my thinking process back then.

Reply:
Our "big" cellmlUnitScaling test has now been replaced with smaller tests that address unit scaling, see our "new" cellmlUnitScalingVoiIndirect, cellmlUnitScalingVoiDirect, cellmlUnitScalingConstant, cellmlUnitScalingState, cellmlUnitScalingStateInitialisation and cellmlUnitScalingRate tests.